includes/reports/overview/data-provider/class-sensei-reports-overview-data-provider-students.php

```diff
@@ -89,6 +97,26 @@ public function get_items( array $filters ): array {
 		return $learners;
 	}
 
+	/**
+	 * Check if last activity filter is enabled.
+	 *
+	 * @return {bolean} Whether last activity filter is enabled.
```

Comment:
Typo:

```suggestion
	 * @return {boolean} Whether last activity filter is enabled.
```

Also probably could just use a type hint instead. :)